example/storybook/src/components/Forms/FormControl/index.stories.mdx

```diff
@@ -275,6 +301,607 @@ The following are the lists of all the libraries and packages the component reli
+                </Radio.Indicator>
+                <Radio.Label lineHeight='$md' color='$text900'>
+                  Monday
+                </Radio.Label>
+              </Radio>
+              <Radio size="sm" value="Apple">
+                <Radio.Indicator>
+                  <Radio.Icon>
+                    <CircleIcon/>
+                  </Radio.Icon>
+                </Radio.Indicator>
+                <Radio.Label lineHeight='$md' color='$text900'>
+                  Tuesday
+                </Radio.Label>
+              </Radio>
+              <Radio size="sm" value="Orange">
+                <Radio.Indicator>
+                  <Radio.Icon>
+                    <CircleIcon/>
+                  </Radio.Icon>
+                </Radio.Indicator>
+                <Radio.Label lineHeight='$md' color='$text900'>
+                  Wednesday
+                </Radio.Label>
+              </Radio>
+            </VStack>
+          </Radio.Group>
+          <FormControl.Error mt='$2'>
+            <FormControl.Error.Text fontSize='$sm' lineHeight='$md'>
+              Choose only one time slot for the meeting
+            </FormControl.Error.Text>
+          </FormControl.Error>
+        </FormControl>
+      `,
+      transformCode: (code) => {
+        return transformedCode(code);
+      },
+      scope: {
+        Wrapper,
+        FormControl,
+        Icon,
+        Radio,
+        WarningIcon,
+        VStack,
+        CircleIcon,
+        Heading,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with Inline Form
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+      <FormControl>
+        <HStack space='md'>
+          <Input>
+            <Input.Input
+              py='$2'
+              type="text"
+              placeholder="Username"
+            />
+          </Input>
+          <Select>
+            <Select.Trigger>
+              <Select.Input placeholder="Country" py='$2'/>
+              <Select.Icon mr="$3">
+                <Icon as={ChevronDownIcon} w='$5' h='$5'/>
+              </Select.Icon>
+            </Select.Trigger>
+            <Select.Portal>
+              <Select.Backdrop />
+              <Select.Content>
+                <Select.DragIndicatorWrapper>
+                  <Select.DragIndicator />
+                </Select.DragIndicatorWrapper>
+                <Select.Item label="Red" value="Red" />
+                <Select.Item label="Blue" value="Blue" />
+                <Select.Item label="Black" value="Black" />
+                <Select.Item label="Green" value="Green" />
+              </Select.Content>
+            </Select.Portal>
+          </Select>
+          <Button px='$3' bg='$darkBlue600'>
+            <Button.Text fontSize='$sm' fontWeight='$medium'>
+              Next
+            </Button.Text>
+          </Button>
+        </HStack>
+        <FormControl.Error mt='$2'>
```

Comment:
This fold is not present in the design. Maybe got copied from above example code. Please remove that

Reply:
updated